test/uxCapture.test.js

```diff
@@ -28,17 +30,16 @@ const onMeasure = jest.fn();
 
 describe('UXCapture', () => {
 	describe('startView', () => {
-		beforeAll(() => {
-			UXCapture.create({ onMark, onMeasure });
```

Comment:
removed the `beforeAll()` setup because it creates an interaction among unit tests that isn't necessary - using `UXCapture.destroy` between tests much more safe